web/js/jquery-3.6.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Committing libraries to the repository is not great, but I used to do it so I understand. I prefer to get them from a CDN as part of the build process

Reply:
> Committing libraries to the repository is not great, but I used to do it so I understand. I prefer to get them from a CDN as part of the build process

Do you have any examples you could share? Or you just use curl to downlaod it before you build?

For some that could work to download from the CDN as they're equal, but I think not for font awesome. In that case I believe the offline version is different.

Reply:
I use urllib to download the files in my setup.py. I'd love to give you examples but I can't at the moment... travel

Reply:
> I use urllib to download the files in my setup.py. I'd love to give you examples but I can't at the moment... travel

No problem, I get the concept. Not sure I want to do that as this won't be installed using pip or any python package manager really. I suppose I could add another script that would be required to run when building.